components/admin/DisputeEvidenceUpload.jsx

```
@@ -0,0 +1,199 @@
+              id="adminNote"
+              placeholder="Provide context or findings regarding this dispute resolution..."
+              value={note}
+              onChange={(e) => setNote(e.target.value)}
+              rows={2}
+              className="text-xs resize-none"
+            />
+          </div>
+
+          {/* File Upload Selector */}
+          <div className="space-y-1">
+            <label className="text-xs font-medium text-muted-foreground">
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)